packages/chain-helpers/src/blockchain/Blockchain.utils.ts

```diff
@@ -142,30 +135,13 @@ async function submitSignedTxErrorMatched(
  */
 export async function submitSignedTx(
```

Comment:
feels like this may only be relevant for our higher level functions, not for external callers? does it make sense to not export it?

Reply:
We call this directly from the Blockchain class when !flag, we need to export it.